dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/DefaultTrackerValidationServiceConfigOrderTest.java

```diff
@@ -73,10 +72,10 @@ void hooksAreExecutedInTrackerValidationConfigOrder()
             .events( Collections.singletonList( event ) )
             .build();
 
-        ValidationReport report = trackerValidationService.validate( bundle );
+        ValidationResult report = trackerValidationService.validate( bundle );
 
         assertTrue( report.hasErrors() );
         assertEquals( 1, report.getErrors().size() );
-        assertEquals( E1048, report.getErrors().get( 0 ).getErrorCode() );
+        assertEquals( E1048.name(), report.getErrors().get( 0 ).getCode() );
```

Comment:
If we have many of these we could add an assertion to our assertion helpers. We have some for the two reports. Maybe we can straighten them after this PR. They were always super close but not quite as ValidationErrorReporter and the other don't share an interface.